### ISSUE TITLE
css in responsive datatables tutorial missing necessary piece

### ISSUE THREAD

**Issue by betsyCC** [davidruvolo51/shinytutorials#41]
Hi David,

I have been working through your tutorial and trying to get the responsive table feature to work and figured out that it wasn't working because the tutorial css was missing this piece below:

```
    .datatable tbody tr th, .datatable tbody tr td {
        display: block;
        padding: 5px 0 5px 12px;
    }
```

I found it here: https://github.com/davidruvolo51/shinyAppTutorials/blob/master/responsive-datatables/www/css/datatable.css and added it to my css file and then it worked...

**Comment by davidruvolo51** [comment]
Hi @betsyCC,

Thanks for the pointing this out. I did not realize it is missing from the tutorial. I’ll add this block to the CSS section. 

Thanks!

**Comment by davidruvolo51** [comment]
Fixed in #42

**Comment by betsyCC** [comment]
BTW - I would have tried to fix it for you and submit a pull request, but then I read the instructions and would have had to download and install a bunch of stuff and was in a time crunch so couldn't do that at the time.  I forked your repo and may try contributing directly in the future though!
